app/blog/page.tsx

```
@@ -1,14 +1,24 @@
-import { FadeIn } from "@/components/animations/fade-in"
-import { BlogList } from "@/components/blog/blog-list"
+import { FadeIn } from "@/components/animations/fade-in";
+import { BlogList } from "@/components/blog/blog-list";
+import { fetchPosts } from "@/lib/wp";
+
+export default async function BlogPage() {
+  const rawPosts = await fetchPosts();
+
+  // Xử lý dữ liệu để đảm bảo các trường là chuỗi
+  const posts = rawPosts.map((post: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)